app/src/test/java/org/simple/clinic/search/results/PatientSearchResultsControllerTest.kt

```diff
@@ -38,196 +28,41 @@ class PatientSearchResultsControllerTest {
-  }
-
-  @Test
-  fun `when a patient search result is clicked, the patient's summary screen should be opened`() {
-    val searchResult = PatientMocker.patientSearchResult()
-    uiEvents.onNext(PatientSearchResultClicked(searchResult))
+    uiEvents.onNext(SearchResultClicked(patientSearchResult))
 
-    verify(screen).openPatientSummaryScreen(searchResult.uuid)
+    verify(screen).openPatientSummaryScreen(patientSearchResult.uuid)
   }
 
   @Test
-  @Parameters(value = [
-    "Abhay",
-    "Vinod"])
-  fun `when create new patient is clicked then patient entry screen should be opened with prefilled name`(
-      name: String
-  ) {
-    val emptySearchResults = PatientSearchResults(
-        visitedCurrentFacility = emptyList(),
-        notVisitedCurrentFacility = emptyList()
-    )
-    whenever(patientRepository.search(any())).thenReturn(Observable.just(emptySearchResults))
-
-    val preFilledEntry = OngoingNewPatientEntry(OngoingNewPatientEntry.PersonalDetails(
-        fullName = name,
-        dateOfBirth = null,
-        age = null,
-        gender = null))
-    whenever(patientRepository.saveOngoingEntry(preFilledEntry)).thenReturn(Completable.complete())
-
-    uiEvents.onNext(PatientSearchResultsScreenCreated(PatientSearchResultsScreenKey(name)))
-    uiEvents.onNext(CreateNewPatientClicked())
-
-    verify(patientRepository).saveOngoingEntry(preFilledEntry)
-    verify(screen).openPatientEntryScreen()
-  }
-
-  @Test
-  fun `when there are only patients in current facility, then "Other Results" header should not be shown`() {
-    val patientSearchResult1 = PatientMocker.patientSearchResult()
-    val patientSearchResult2 = PatientMocker.patientSearchResult()
-
-    val patientSearchResults = PatientSearchResults(
-        visitedCurrentFacility = listOf(patientSearchResult1, patientSearchResult2),
-        notVisitedCurrentFacility = emptyList()
-    )
-    whenever(patientRepository.search(any())).thenReturn(Observable.just(patientSearchResults))
-
-    uiEvents.onNext(PatientSearchResultsScreenCreated(PatientSearchResultsScreenKey("name")))
-
-    verify(patientRepository).search("name")
-    verify(screen).updateSearchResults(listOf(
-        InCurrentFacilityHeader(facilityName = currentFacility.name),
-        PatientSearchResultRow(
-            searchResult = patientSearchResult1,
-            currentFacility = currentFacility,
-            phoneObfuscator = phoneObfuscator,
-            dateOfBirthFormat = dateOfBirthFormatter,
-            clock = utcClock,
-            userClock = userClock
-        ),
-        PatientSearchResultRow(
-            searchResult = patientSearchResult2,
-            currentFacility = currentFacility,
-            phoneObfuscator = phoneObfuscator,
-            dateOfBirthFormat = dateOfBirthFormatter,
-            clock = utcClock,
-            userClock = userClock
+  fun `when RegisterNewPatient event is emitted open PatientSummaryScreen`() {
```

Comment:
Rename this to `when register new patient is clicked, then patient entry screen must be opened`?

Reply:
done